.ci/linux-steps.yaml

```diff
@@ -23,16 +23,17 @@ steps:
 
     sudo apt-get install -y --allow-unauthenticated libopenblas-dev liblapack-dev g++ libboost1.70-dev libarmadillo-dev xz-utils
 
-    if [ '$(python.version)' == '2.7' ]; then
-      sudo apt-get install -y --allow-unauthenticated python-pip cython python-numpy python-pandas
-      sudo pip install --upgrade --ignore-installed setuptools cython
-      sudo pip install "zipp<3.1"
-    fi
+    sudo add-apt-repository ppa:deadsnakes/ppa
+    sudo apt-get update
+    sudo apt update
+    sudo apt install python3.7
+    sudo apt-get install -y --allow-unauthenticated python3-pip python3-numpy
 
-    if [ '$(python.version)' == '3.7' ]; then
-      sudo apt-get install -y --allow-unauthenticated python3-pip cython3 python3-numpy
-      sudo pip3 install --upgrade --ignore-installed setuptools cython pandas
-    fi
+    python3.7 --version
+    python3 --version
+    sudo python3.7 -m pip install "Cython>0.24"
```

Comment:
Maybe we need to use `/opt/hostedtoolcache/Python/3.7.7/x64/bin/python3` here instead?  I wonder if this is just installing Cython to the wrong Python installation.  Just an idea... maybe the issue is trickier than that. :+1:

Reply:
Thanks @rcurtin, That did work. We should get the binding test in linux machine after the build finishes.

Reply:
I think this is ready to go now. Thanks.